hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -162,6 +176,7 @@ public Response getKeysForContainer(
     Map<String, KeyMetadata> keyMetadataMap = new LinkedHashMap<>();
     long totalCount;
     try {
+      prevKeyPrefix = buildObjectPathForFileSystemBucket(prevKeyPrefix);
```

Comment:
We are already getting FSO / Legacy inside, same can be retrieved outside, no need iterate for those table.

Reply:
Thank you for your message, Sumit. However, in the code, we will iterate through the `KeyTable` and `FileTable` for each `containerKeyPrefix` present in the container. Each container contains a mixture of keys from both the KeyTable and the FileTable. Therefore, the `correctPathForLayout` method will only be called once at the beginning, and setting the layout to either `FSO` or `LEGACY` at the start will not display keys from the other table.